src/Overlay.tsx

```diff
@@ -9,7 +9,7 @@ import {useCombinedRefs} from './hooks/useCombinedRefs'
 
 type StyledOverlayProps = {
   width?: keyof typeof widthMap
-  height?: keyof typeof heightMap
+  height?: string
```

Comment:
This could be switched back now that we have `initial` in the map, right?

Reply:
Yup, that change is no longer necessary—we can realign the internal (`StyledOverlayProps`) and external (`OverlapProps`) contracts.

Incidentally, the reason isn’t because `initial` is included in `heightMap`, but rather because [imperatively setting `height`](https://github.com/primer/components/blob/c54c7e5b8e025983dd8190de9d50d356ab55e924/src/Overlay.tsx#L118) means we are no longer passing arbitrary `height` values to the `StyledOverlay` component, so the type of `height` no longer needs to be widened from `keyof typeof heightMap` to `string`.

Reply:
Fixed in https://github.com/primer/components/pull/1287/commits/eb770c82b89c2d7659dd5f6fd06dfe50a80f368c

Reply:
Sidebar: Do you think CSP would block the imperative approach? If so, we could restore the original approach (in https://github.com/primer/components/pull/1287/commits/eb298ad5d7cbacfb9b26d77cdf01d54ebf420f53).